sa/sa.go

```diff
@@ -1798,14 +1808,23 @@ func (ssa *SQLStorageAuthority) RevokeCertificate(ctx context.Context, req *sapb
 // cert is already revoked, if the new revocation reason is `KeyCompromise`,
 // and if the revokedDate is identical to the current revokedDate.
 func (ssa *SQLStorageAuthority) UpdateRevokedCertificate(ctx context.Context, req *sapb.RevokeCertificateRequest) (*emptypb.Empty, error) {
-	if req.Serial == "" || req.Date == 0 || req.Backdate == 0 || req.Response == nil {
+	if req.Serial == "" || req.Date == 0 || req.Backdate == 0 {
+		return nil, errIncompleteRequest
+	}
+	if req.Response == nil && !features.Enabled(features.ROCSPStage6) {
 		return nil, errIncompleteRequest
 	}
 	if req.Reason != ocsp.KeyCompromise {
 		return nil, fmt.Errorf("cannot update revocation for any reason other than keyCompromise (1); got: %d", req.Reason)
 	}
+
 	thisUpdate := time.Unix(0, req.Date)
 	revokedDate := time.Unix(0, req.Backdate)
+	ocspResponse := req.Response
+	if features.Enabled(features.ROCSPStage6) {
+		ocspResponse = nil
+	}
```

Comment:
I don't believe this is necessary. We already know this is nil (and that's fine) based on the check on line `1814`.

Reply:
Same as above: we know that it _can_ be nil, not that it definitely is.